Resources/template.json

```diff
@@ -282,7 +299,43 @@
       ],
       "copy": {
         "name": "keyvaultReadAccessCopy",
-        "count": "[length(parameters('defaultKeyVaultReadAccess'))]",
+        "count": "[variables('defaultKeyVaultReadAccessHack')]",
+        "mode": "Serial",
+        "batchSize": 1
+      }
+    },
+    {
+      "name": "[concat('sharedKeyVaultCertificateWriteAccessPolicy-', copyIndex())]",
+      "type": "Microsoft.Resources/deployments",
+      "condition": "[greater(length(parameters('certificateKeyVaultWriteAccess')), 0)]",
+      "apiVersion": "2017-05-10",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('dfcDevOpsTemplatesUrl'),'/KeyVault/keyvault-access-policy.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "keyVaultName": {
+            "value": "[variables('keyVaultName')]"
+          },
+          "servicePrincipalObjectId": {
+            "value": "[parameters('certificateKeyVaultWriteAccess')[copyIndex()]]"
+          },
+          "certificatePermissions": {
+            "value": [
+              "create",
+              "update"
+            ]
+          }
+        }
+      },
+      "dependsOn": [
+        "sharedKeyVault"
+      ],
+      "copy": {
+        "name": "certificateKeyVaultWriteAccessCopy",
+        "count": "[variables('certificateWriteAccessHack')]",

```

Comment:
Isn't this variable a bool rather than an int?

Reply:
This works identically to the above